dokka-runners/dokka-gradle-plugin/src/main/kotlin/dependencies/FormatDependenciesManager.kt

```diff
@@ -107,15 +104,15 @@ class FormatDependenciesManager(
     private val dokkaPublicationPluginClasspath: NamedDomainObjectProvider<Configuration> =
         project.configurations.register(configurationNames.publicationPluginClasspath) {
             description =
-                "Dokka Plugins classpath for a $formatName Publication (consisting of 1+ Dokka Module)."
+                "$INTERNAL_CONF_DESCRIPTION_TAG Dokka Plugins classpath for a $formatName Publication (consisting of 1+ Dokka Module)."
```

Comment:
Q: should it be with internal tag? AFAIU it's user-visible configuration to add plugins to publication?

Reply:
Good spot, you're right, thanks!